test/util/param_spec_test.js

```diff
@@ -0,0 +1,79 @@
+import ParamSpec from "../../src/js/util/param_spec"
+
+describe('ParamSpec', function () {
+  context('with clamping', function () {
+    beforeEach(function () {
+      this.subject = ParamSpec(0, 127, 0, 10)
+    })
+
+    describe('map', function () {
+      it('maps a value to the distribution', function () {
+        expect(this.subject.map(63.5)).to.eql(5)
+      })
+
+      it('maps a value to the distribution', function () {
+        expect(this.subject.map(0)).to.eql(0)
+      })
+
+      it('maps a value to the distribution', function () {
```

Comment:
Why does 127 become 10? sorry i'm just really confused!

Don't we just have sliders from 0 - 127? What is distribution for?